### PR TITLE
Add page "How to Repair"

### PR DESCRIPTION
**Pull request by monsieurhannes**

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -6,6 +6,7 @@ title: Repair Café Malmö
 navigation:
   - {name: 'Home', link: '/'}
   - {name: 'About Us', link: '/about'}
+  - {name: 'Cheat Sheet', link: '/cheatsheet'}
```
Comment (ulfwin) [inline _config.yml]:
```suggestion
  - {name: 'How To Repair', link: '/howtorepair'}
```
Actually I changed the name here as well

**Comment by ulfwin** [comment]
Seems to be working. Pretty ugly formatting I would say though... someone else wants to take a look at that, or merge to master for now?

**Comment by monsieurhannes** [comment]
One reason why I added chapters instead of the bullet points, they allow linking. That way we can link within a document (e.g. https://deploy-preview-32--repaircafe.netlify.com/howtorepair/#search-for-repair-solutions-on-the-internet)

**Comment by ulfwin** [comment]
I chose an ordered list since the steps are meant to be followed in order. Is there a reason to not do that? Can we add it back?

**Comment by monsieurhannes** [comment]
> 
> 
> I chose an ordered list since the steps are meant to be followed in order. Is there a reason to not do that? Can we add it back?

Sure, I don't think it's bad to have ordered lists. But I think we should keep the headers to make it linkable and more structured :) So I could imagine sth like this: `### 1. Tralallaa`

**Comment by qerub** [comment]
Good teamwork! 👏🏻

**Comment by ulfwin** [comment]
Yep, worked great! 😄

**Pull request merged**